spec/lib/requirements/file_attachment_metadata_checker_spec.rb

```diff
@@ -1,14 +1,23 @@
 RSpec.describe Requirements::FileAttachmentMetadataChecker do
-  let(:checker) { described_class.new }
-
   describe "#pre_publish_issues" do
+    let(:document_type) { build :document_type, attachments: "featured" }
+
     it "returns no issues if there are none" do
+      checker = described_class.new(build(:edition, document_type: document_type))
       attachment_revision = build(:file_attachment_revision, official_document_type: "unofficial")
       issues = checker.pre_publish_issues(attachment_revision)
       expect(issues).to be_empty
     end
 
+    it"returns no issues unless the document type supports featured attachments" do
+      checker = described_class.new(build(:edition))
+      attachment_revision = build(:file_attachment_revision)
+      issues = checker.pre_publish_issues(attachment_revision)
+      expect(issues).to be_empty
+    end
```

Comment:
I'd move this test below the next one, so that it's clearer that the absence of `official_document_type` would ordinarily raise issues.

Reply:
That wouldn't make much difference to me. I think I used to put effort into making tests tell a story, but then I realised I seldom read them top-to-bottom. Since this is causing a bug on the master branch, could we postpone this discussion?

Reply:
Just spotted a formatting issue too - perhaps you could reorder at the same time as fixing that? 😇 
It's not a deal-breaker though, I appreciate wanting to fix this quickly.